docs/src/fees.md

```diff
@@ -1,3 +1,8 @@
+---
+sidebar_position: 5.1
```

Comment:
I'm not sure how `sidebar_position` is used correctly, but in other places we seem to use plain numbers, so should this be `5` instead of `5.1`?